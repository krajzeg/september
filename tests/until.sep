until := |?condition, body| {
	while(!(condition.resolve())) {
		body.resolve();
	};
};

print("This should print values from 1 to 5 and nothing else.");

i := 1;
until (i > 5) {
	print(i);
	i = i + 1;	
};

until (True) {
	print("This should not be printed.");
};
