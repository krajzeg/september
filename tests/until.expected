This should print values from 1 to 5 and nothing else.
1
2
3
4
5
